ksql-common/src/main/java/io/confluent/ksql/util/SchemaUtil.java

```diff
@@ -83,30 +84,65 @@
           .put(Schema.Type.FLOAT64, Schema.OPTIONAL_FLOAT64_SCHEMA)
           .build();
 
+  private static final ImmutableMap<String, String> TYPE_MAP =
+      new ImmutableMap.Builder<String, String>()
+          .put("STRING", "VARCHAR(STRING)")
+          .put("INT64", "BIGINT")
+          .put("INT32", "INTEGER")
+          .put("FLOAT64", "DOUBLE")
+          .put("BOOLEAN", "BOOLEAN")
+          .put("ARRAY", "ARRAY")
+          .put("MAP", "MAP")
+          .put("STRUCT", "STRUCT")
+          .build();
+
+  private static final Map<Schema.Type, Function<Schema, Type>> SCHEMA_TYPE_TO_JAVA_TYPE =
+      ImmutableMap.<Schema.Type, Function<Schema, Type>>builder()
```

Comment:
nit: why does this need to be a function (`s` is never used)?

Reply:
Same answer from above. Once I introduce the Decimal type, the BYTES schema will need a function that verifies if schema.name() is a Decimal logical type, otherwise throw an error.